assets/terraform-modules/bare-metal/flatcar-linux/kubernetes/bootkube.tf

```diff
@@ -3,8 +3,8 @@ module "bootkube" {
   source = "../../../bootkube"
 
   cluster_name                    = var.cluster_name
-  api_servers                     = [var.k8s_domain_name]
```

Comment:
Please commit workers change and controllers change separately, as they have different impact.